ofp4/backend.cpp

```diff
@@ -625,6 +622,86 @@ class FlowGenerator : public Inspector {
         declarations->push_back(makeFlowRule(flowRule, cfgtable->table->externalName()));
     }
 
+    void convertKey(CFG::TableNode* table,
+                    IR::Vector<IR::DDlogMatchCase>* tableCases,
```

Comment:
this seems to be an "out" argument, maybe you want to add a comment.
I think this function deserves a comment too, ideally describing the parameters.

Reply:
You are right. This needs some comments. I added some. They probably help.